core/file.c

```diff
@@ -650,7 +650,8 @@ int parse_txt_file(const char *filename, const char *csv)
 		cur_cylinder_index++;
 
 		lineptr = strstr(memtxt.buffer, "Dive started at");
-		while (!empty_string(lineptr) && (lineptr = strchr(lineptr, '\n')) && ++lineptr) {
+		while (!empty_string(lineptr) && (lineptr = strchr(lineptr, '\n'))) {
+			++lineptr;	// Skip over '\n'
```

Comment:
seems like a semi-bug fix even?

Reply:
Hmmm... Now that you say it, perhaps this was originally intended as
```
while (!empty_string(lineptr) && (lineptr = strchr(lineptr, '\n')) && *++lineptr)
```
?
In any case, `*lineptr == '\0'` is handled gracefully in `next_mkvi_key()`, so not an issue...

Reply:
`*++lineptr` was probably was the intended.